backbone/src/main/javaorg/researchstack/backbone/DataProvider.java

```diff
@@ -160,6 +188,15 @@ public static void init(DataProvider instance)
     public abstract void saveConsent(Context context, TaskResult consentResult);
```

Comment:
My initial thought was to deprecate this, since it encourages coupling the DataProvider on the StepLayouts. What do you guys think?